packages/@n8n/permissions/src/scope-information.ts

```diff
@@ -0,0 +1,24 @@
+import { RESOURCES } from './constants.ee';
+import type { Scope, ScopeInformation } from './types.ee';
+
+function buildResourceScopes() {
+	const resourceScope = Object.keys(RESOURCES).flatMap((resource) => {
+		const operations = RESOURCES[resource as keyof typeof RESOURCES];
+		const resourceScope = operations.map(
+			(operation) => `${resource}:${operation}` as const,
+		) as string[];
+		resourceScope.push(`${resource}:*` as const); // Add wildcard for all operations on this resource
+		return resourceScope;
+	}) as Scope[];
+	resourceScope.push('*' as const); // Add wildcard scope
+	return resourceScope;
+}
+
+export const ALL_SCOPES = buildResourceScopes();
+
+export const scopeInformation: Partial<Record<Scope, ScopeInformation>> = {
```

Comment:
Is that really necessary ?

Reply:
This is just a place holder to describe the scopes for the UI. So I think we should fill this list at some point, when we research all scopes and what they are used for.